nodeadm/internal/daemon/systemd.go

```diff
@@ -55,7 +55,7 @@ func (m *systemdDaemonManager) GetDaemonStatus(name string) (DaemonStatus, error
 	if err != nil {
 		return DaemonStatusUnknown, err
 	}
-	switch status.Value.String() {
+	switch status.Value.Value().(string) {
```

Comment:
Not understanding this change. If our existing code was mishandling this, wouldn't we always get `DaemonStatusUnknown` back?

Reply:
we didn't have any callers of `GetDaemonStatus` before this, so we didn't *mishandle* it, but it just never worked 🫠